src/main/java/doit/jpastudy2/repository/GradesRepository.java

```diff
@@ -0,0 +1,8 @@
+package doit.jpastudy2.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface GradesRepository extends JpaRepository {
```

Comment:
```suggestion
public interface GradesRepository extends JpaRepository<Grades, Long> {
```

JpaRepository를 사용하실 떄는 제네릭 부분에 <Entity 클래스 명, PK의 타입> 형태로 작성해주셔야합니다!